Models/GebruikersModel.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+class gebruikersModel extends Models {
+    public static $_tableName = "gebruikersModel";
+
+     protected id: int
```

Comment:
Helaas niet geldige php code, ik heb tot nu toe nog niet veel typing gebruikt maar misschien wel cool om te doen. Zo doe je het in php: https://stitcher.io/blog/typed-properties-in-php-74. Lines moeten ook nog altijd worden afgesloten met `;`

Reply:
Daarnaast gebruik je altijd dollarteken bij variablen